lib/player.js

```diff
@@ -1408,14 +1408,15 @@ shaka.Player.prototype.getStats = function() {
  *
  * @param {string} uri
  * @param {string} language
+ * @param {?string} label
  * @param {string} kind
  * @param {string} mime
  * @param {string=} opt_codec
  * @return {!Promise.<shakaExtern.Track>}
  * @export
  */
 shaka.Player.prototype.addTextTrack = function(
-    uri, language, kind, mime, opt_codec) {
+    uri, language, label, kind, mime, opt_codec) {
```

Comment:
Please do not add additional parameters to the middle of an exported method.  This would break backward compatibility.  Instead, make it an optional parameter after opt_codec.